packages/web-components/src/keplr-connection/watchWallet.js

```diff
@@ -6,18 +6,15 @@ import { assertHasData } from '@agoric/smart-wallet/src/utils';
 import { Errors } from '../errors.js';
 import { queryBankBalances } from '../queryBankBalances.js';
 
-// Import ambient types
-import '@agoric/ertp/exported.js';
```

Comment:
In typescript this doesn't work. We don't rely on the types anyway so I changed to unknown.